webui/react/.storybook/preview.js

```diff
@@ -1,8 +1,27 @@
-import 'shared/styles/index.scss';
-import 'shared/styles/storybook.scss';
```

Comment:
' or " ? let's use whichever we use elsewhere

Reply:
We should define the rule in ESLint.

Reply:
Fixed!